models/src/main/java/com/vimeo/networking2/Upload.kt

```diff
@@ -40,7 +40,7 @@ data class Upload(
      */
     @Internal
```

Comment:
Are we still planning to use the `@Internal` annotation?

Reply:
Yes